deploy/0.9.0/contracts.json

```diff
@@ -0,0 +1,11 @@
+{
+  "version": "0.9.0",
+  "networks": {
+    "mainnet": {
+      "AddressListFactory": "0x27a1E374B7176341D700A8e1AF8e1b36908E3d09"
```

Comment:
Need to fix name of contract :)

Reply:
Oops. Sorry about that. copy paste error.